public/en/releases.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
```

Comment:
Not yet, but I'm open to us getting there.

Regardless: this specific line is tied to https://github.com/iojs/website/blob/264_build_process_3/source/templates/main.html so a PR against that file would fix this across the entire site. (This file was generated like everything in `public`.)